test/fixtures/relationships.yml

```diff
@@ -0,0 +1,17 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/Fixtures.html
+
+one:
```

Comment:
Wow, I didn't know you could do fixtures like this. You learn something every day! Now I want to go back in and clean mine up to not use uids.